src/CoreBundle/Resources/public/javascript/table.js

```diff
@@ -149,6 +149,8 @@ $(document).ready(function () {
 
     var searchLookup = $('select#searchLookup');
 
+    let searchInputFieldCleared = true;
```

Comment:
Is "input field cleared" the default / main usage case?

I'd expect the values to be swapped: set it to true in select2:clear below.

Reply:
Yes, you're right. Changed.